src/api/python/CMakeLists.txt

```diff
@@ -25,7 +25,10 @@ check_python_module("setuptools")
 execute_process(
   COMMAND
     ${Python_EXECUTABLE} -c "from __future__ import print_function; \
-      import os; import skbuild; \
+      import os; import sys; \
+      sys.stdout = open(os.devnull, 'w'); \
+      import skbuild; \
+      sys.stdout = sys.__stdout__; \
```

Comment:
This hack is needed because importing skbuild prints to stdout in some versions, which stores the wrong value to the `OUTPUT_VARIABLE`. Once scikit is removed, this whole command will be removed too.

Reply:
Same here.